### ISSUE TITLE
Scalar ID: not being possible to define custom generator 

### ISSUE THREAD

**Issue by cesarsalesgomes** [ardeois/graphql-codegen-typescript-mock-data#91]
### Describe the Bug

Custom generator defined for Scalar ID seems to be always returning string type:

>

```yml
./cat/cat.mock.ts:
  schema: http://localhost/graphql
  documents:
    - ./cat/queries/*.graphql
  plugins:
    - typescript-mock-data:
        typesFile: "./cat.sdk.ts"
        scalars:
          ID:
            generator: integer
            arguments:
              - 1
              - 100
```

### Generated mock

>

```typescript
export const aCat = (overrides?: Partial<Cat>): Cat => {
  return {
    id:
      overrides && overrides.hasOwnProperty("id")
        ? overrides.id!
        : "f54c0ef5-82b1-4327-8142-8bf4cf1f3a2a",
  };
};
```

>

### What version of graphql-codegen-typescript-mock-data are you using?

> 2.2.5

### What version of Node.js are you using?

> 16.14.0

**Comment by MH4GF** [comment]
related: https://github.com/ardeois/graphql-codegen-typescript-mock-data/issues/89#issuecomment-1255885159

**Comment by ardeois** [comment]
Should be fixed in #95
